### PR TITLE
Added HCL examples to service discovery page

### PR DESCRIPTION
**Pull request by preetapan**
Added HCL examples to service discovery page - I updated the main service discovery example as well as the multiple services example in this PR.

### CODE REVIEW

File: website/content/docs/discovery/services.mdx [hunk 1]
```diff
@@ -32,7 +32,91 @@ Send a `SIGHUP` to the running agent or use [`consul reload`](/commands/reload)
...
+
+  port                = 8000
+  socket_path         = "/tmp/redis.sock"
+  enable_tag_override = false
+
+  checks = [
+    {
+      args     = ["/usr/local/bin/check_redis.py"]
+      interval = "10s"
+    },
```
Comment (karl-cardenas-coding) [inline website/content/docs/discovery/services.mdx:71]:
```suggestion
    }
```

File: website/content/docs/discovery/services.mdx [hunk 2]
```diff
@@ -32,7 +32,91 @@ Send a `SIGHUP` to the running agent or use [`consul reload`](/commands/reload)
...
+    expose = {
+      checks = true
+
+      paths = [
+        {
+          path            = "/healthz"
+          local_path_port = 8080
+          listener_port   = 21500
+          protocol        = "http2"
+        },
```
Comment (karl-cardenas-coding) [inline website/content/docs/discovery/services.mdx:102]:
```suggestion
        }
```

**Review by karl-cardenas-coding — APPROVED** [review]
Looks great! I caught two commas in the first HCL example.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/544655.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 295ab0b3a5dac411ae36ae3ad80028c00ca2297e onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 295ab0b3a5dac411ae36ae3ad80028c00ca2297e onto `release/1.11.x` succeeded!

**Pull request merged**